cmd/flag/debug_flag.go

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright © 2019 Infostellar, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flag
+
+import "github.com/spf13/cobra"
+
+type DebugFlag struct {
+	IsDebug bool
+}
+
+// Add flags to the command.
+func (f *DebugFlag) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().BoolVarP(&f.IsDebug, "debug", "", false, "Output debug information. (default false)")
```

Comment:
I wonder if 'verbose' is a better name for this flag? 'debug' sounds like an internal thing for our use.

It might be better to have a description that describes what extra information will be output (at least give some examples).

Reply:
We've already had verbose flag which shows antenna status in every seconds. While antenna status is useful for users to see such information like elevation, showing framing type of each packet is too detail and doesn't give much values except for users who want to check their framing option in command line. That is the reason I added the debug flag separate for verbose flag. Probably, it should be implemented as a "log level" in the future.